kogito-scripts-quarkus/README.md

```diff
@@ -59,7 +71,7 @@ To make use of this application it is as simple as putting a sending request to
 
```

Comment:
Forgot here about swagger-ui:

line 65
`(Dev Mode Only) https://quarkus.io/guides/openapi-swaggerui#use-swagger-ui-for-development`

Reply:
fixed @radtriste